test/variable.xspec

```diff
@@ -173,6 +170,25 @@
 			<x:expect label="reflects value redefined within same scenario."
 				select="'value #2 (redefined using same prefix or different prefix)'"/>
 		</x:scenario>
+		<x:scenario label="Redefinition of variable and mixed notations for its name">
+			<x:variable as="element(variable-child)"
+				name="
+				&#x09;&#x0A;&#x0D;&#x20;
+				Q{http://example.org/ns/my/variable}var
+				&#x09;&#x0A;&#x0D;&#x20;"
```

Comment:
Whitespaces were inserted into `eqname.xspec` by #803.
Even after this test relocation, a local variable name with whitespaces is still covered by [`eqname.xspec`](https://github.com/galtm/xspec/blob/1f87de3b37ed5ef14620ff3201d7bb385f3dbdbe/test/eqname.xspec#L101-L104). So I think we can remove the whitespaces from `variable.xspec`:

```suggestion
				name="Q{http://example.org/ns/my/variable}var"
```

Reply:
Fixed with df7e559203cf3a052059d8f6d6f91fa5f733e329